rollup.config.js

```diff
@@ -1,16 +1,11 @@
 import esbuild from "rollup-plugin-esbuild";
-import nodePolyfills from "rollup-plugin-polyfill-node";
 import { nodeResolve } from "@rollup/plugin-node-resolve";
 import commonjs from "@rollup/plugin-commonjs";
 
 const input = "./src/index.ts";
 const plugins = [
-  nodeResolve({
-    preferBuiltins: false,
-    browser: true,
-  }),
+  nodeResolve({ preferBuiltins: false, browser: true }),
   commonjs(),
-  nodePolyfills(),
```

Comment:
Nice 🙌  are we sure we can remove these already, i.e. aren't they still needed for pino for example?

Reply:
I tested dapp example with this version and all loggs are still there, I assume thats pino working yeh?

Reply:
Ok let's go for it 👍